src/editor/components/flow/edit-dialog.tsx

```diff
@@ -19,7 +19,7 @@ const states = [
 ]
 
 export const EditDataFlowDialog = ( { isEditOpen, setIsEditOpen, save, flow, setFlow, deleteFlow }: EditDataFlowDialogProps ): JSX.Element => {
-  const [deleteFlowName, setDeleteFlowName] = useState()
+  const [deleteFlowName, setDeleteFlowName] = useState( '' )
```

Comment:
Why was this necessary? You didn't change anything else in this file?

Reply:
Types. I changed it to `useState<string>()` – which you said you preferred.